src/assets/i18n/en.json5

```diff
@@ -6787,4 +6787,6 @@
   "live-region.ordering.dropped": "{{ itemName }}, dropped at position {{ index }} of {{ length }}.",
 
   "dynamic-form-array.sortable-list.label": "Sortable list",
+
+  "embargo.listelement.badge": "embargo until {{ date }}",
```

Comment:
Can you capitalize this?

Reply:
Yes, like this? Embargo until {{ date }}